packages/dev/test/penalizer/PenalizerService.test.ts

```diff
@@ -40,7 +40,8 @@ contract('PenalizerService', function (accounts) {
       url: '',
       workdir: '',
       etherscanApiUrl: 'etherscanApiUrl',
-      relayHubAddress: env.relayHub.address
+      relayHubAddress: env.relayHub.address,
```

Comment:
yes, its required... how did the test pass without it ?

Reply:
It was given default owner (zero address), which is now no longer allowed. This is from previous PR #665